packages/manager/apps/zimbra/src/pages/dashboard/GeneralInformation.tsx

```diff
@@ -1,17 +1,56 @@
 import React from 'react';
-import { OsdsTile } from '@ovhcloud/ods-components/react';
+import { useEnvironment } from '@ovh-ux/manager-react-shell-client';
+import { OsdsTile, OsdsDivider } from '@ovhcloud/ods-components/react';
+import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';
+
+import { LinkType, Links, Subtitle } from '@ovhcloud/manager-components';
+import { GUIDES_LIST } from '@/guides.constants';
 
 function GeneralInformation() {
+  const guidesHeadersCategory = 'zimbra_user_guide';
+  const { ovhSubsidiary } = useEnvironment().getUser();
+  const guideList = GUIDES_LIST.user_guide[guidesHeadersCategory];
+  const webmail = GUIDES_LIST.webmail.url;
+
   return (
     <div className="grid xs:grid-cols-1 sm:grid-cols-2 md:grid-cols-3 py-6">
       <div className="p-3">
         <OsdsTile>Tile 1</OsdsTile>
       </div>
-      <div className="p-3">
+      <div className="p-3 relative">
         <OsdsTile>Tile 2</OsdsTile>
       </div>
       <div className="p-3">
-        <OsdsTile>Tile 3</OsdsTile>
+        <OsdsTile className="w-full h-full flex-col">
+          <div className="flex flex-col w-full">
+            <span slot="start">
+              {/* title and label to define with translations */}
+              <Subtitle>Useful Links</Subtitle>
+              <OsdsDivider separator />
+              <Links
+                type={LinkType.external}
+                target={OdsHTMLAnchorElementTarget._blank}
+                href={webmail}
+                label="webmail access"
+                className="block"
+              />
+              <OsdsDivider separator />
+              <Links
+                type={LinkType.external}
+                label="Administrator's Guide"
+                className="block"
+              />
+              <OsdsDivider separator />
+              <Links
+                type={LinkType.external}
+                href={guideList.url[ovhSubsidiary] || guideList.url.DEFAULT}
+                target={OdsHTMLAnchorElementTarget._blank}
+                label="User Guide"
+                className="block"
+              />
```

Comment:
Maybe we can do a map for avoid a lot html code.
For example (have to be adapted):

```suggestion
Object.values(GUIDES_LIST).map((guide, index) => 
           <div key={index}>
              <OsdsDivider separator />
              <Links
                type={LinkType.external}
                target={OdsHTMLAnchorElementTarget._blank}
                href={guide.webmail}
                label={t(guide.translationKey)
                className="block"
              />
           </div>
```